### DISCUSSION TITLE
Web clipper for Firefox on Android?

### DISCUSSION THREAD

**Discussion by kjgcoop** [TriliumNext/Trilium#4292] [General]
Trillium is great! I have the Web clipper installed on my desktop. Do you have any plans to implement it for Firefox on Android?

Thank you!

**Comment by zadam** [comment]
As I understand, extensions are still not completely open on Firefox for Android, so I will wait until that changes.

Otherwise it should support the same webextension standard, so the clipper should work (almost) out of the box.

**Reply by kjgcoop** [reply]
Awesome, thanks for the update!
